test/test_distributions.py

```diff
@@ -1731,7 +1749,7 @@ def test_entropy_shape(self):
                     continue
 
     def test_bernoulli_shape_scalar_params(self):
-        bernoulli = Bernoulli(0.3)
+        bernoulli = Bernoulli(0.3, validate_args=True)
```

Comment:
It would be easier to define `Distribution.set_default_valide_args(True)` in `setUp()` (and correspondingly turn it off in `tearDown()`

Reply:
Based on the global initialization, I removed the `set_default_args_validation` parameter, which is why I manually set it for each case.

Reply:
Sorry I don't understand. Wouldn't it be simpler to pull in the `set_default_validate_args()` function from #134 and set that in a `setUp` method, rather than set it manually?